test/app/core/utils.spec.coffee

```diff
@@ -120,76 +120,259 @@ describe 'Utility library', ->
       levelNumberMap = utils.createLevelNumberMap(levels)
       expect((val.toString() for key, val of levelNumberMap)).toEqual(['1', '1a', '1b', '1c', '2', '2a', '2b', '3', '4', '4a', '5'])
 
-  describe 'findNextlevel', ->
-    # r=required p=practice c=complete *=current
+  describe 'find next level and assessment', ->
```

Comment:
Name this test `findNextLevel and findNextAssessmentForLevel` since it's testing those two functions. Also, it'd be good if there were some sort of human-readable description of the expected behavior. So for example, could update the test descriptions. But probably what would be easiest would be to have a description here in the comment block. We have something like that in the code itself, but since this is a "spec" file, would probably be more appropriate here.